tests/store/test_sqlalchemy_store.py

```diff
@@ -69,46 +76,40 @@ def test_default_experiment(self):
         self.assertEqual(first.name, "Default")
 
     def test_default_experiment_lifecycle(self):
-        with TempDir(chdr=True) as tmp:
-            tmp_file_name = "sqlite_file_to_lifecycle_test_{}.db".format(int(time.time()))
-            self._setup_database("/" + tmp.path(tmp_file_name))
-
-            default_experiment = self.store.get_experiment(experiment_id=0)
-            self.assertEqual(default_experiment.name, Experiment.DEFAULT_EXPERIMENT_NAME)
-            self.assertEqual(default_experiment.lifecycle_stage, entities.LifecycleStage.ACTIVE)
-
-            self._experiment_factory('aNothEr')
-            all_experiments = [e.name for e in self.store.list_experiments()]
-            six.assertCountEqual(self, set(['aNothEr', 'Default']), set(all_experiments))
+        default_experiment = self.store.get_experiment(experiment_id=0)
```

Comment:
The changes to this test should be a no-op (I removed `with TempDir`, which unfortunately shifted everything else up a line / removed one level of indentation in the diff)